dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerMain.java

```diff
@@ -98,12 +99,15 @@ public static void main(String[] arguments)
                       symbolProblems, classFile -> graph.isReachable(classFile.getBinaryName())));
         }
 
-        System.out.println(SymbolProblem.formatSymbolProblems(symbolProblems));
-
         Path writeAsExclusionFile = linkageCheckerArguments.getOutputExclusionFile();
         if (writeAsExclusionFile != null) {
           ExclusionFiles.write(writeAsExclusionFile, symbolProblems);
           System.out.println("Wrote the linkage errors as exclusion file: " + writeAsExclusionFile);
+          return;
+        }
+
+        if (!symbolProblems.isEmpty()) {
+          System.out.println(SymbolProblem.formatSymbolProblems(symbolProblems));
         }
```

Comment:
I know think with "--output-exclusion-file" option the tool should not print linkage errors. Showing "Wrote the linkage errors as exclusion file: ..." message is the exact effect of the option. Showing other linkage errors in console, potentially hundreds of lines, obscures the message.